libraries/oneview_resource_base.rb

```diff
@@ -36,54 +38,77 @@ def create_or_update(item = nil)
           Chef::Log.info("#{resource_name} '#{name}' is up to date")
         else
           Chef::Log.debug "#{resource_name} '#{name}' Chef resource differs from OneView resource."
-          Chef::Log.info "Update #{resource_name} '#{name}'"
-          converge_by "Update #{resource_name} '#{name}'" do
-            item.update(temp) # Note: Assumes resources supports #update
+          Chef::Log.info "#{method_2.to_s.capitalize} #{resource_name} '#{name}'"
+          converge_by "#{method_2.to_s.capitalize} #{resource_name} '#{name}'" do
+            item.update(temp)
```

Comment:
we don't have an edit method....update was kept the same. It was just something to fix the message